src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -211,7 +214,7 @@ public static void createTargetGCSBucketWithCSVFile() throws IOException, URISyn
     BeforeActions.scenario.write("GCS target bucket name - " + gcsTargetBucketName);
   }
 
-  @After(order = 1, value = "@GCS_SINK_TEST or @GCS_SINK_EXISTING_BUCKET_TEST")
+  @After(order = 1, value = "@GCS_SINK_TEST or @GCS_SINK_EXISTING_BUCKET_TEST or @GCS_CSV_MULTI_PART_UPLOAD")
```

Comment:
replace CSV text with SINK text in hook name

Reply:
Changes Added as per the Suggestion.